custom/ilsgateway/tasks.py

```diff
@@ -17,14 +18,24 @@
 from custom.ilsgateway.models import ILSGatewayConfig, SupplyPointStatus, DeliveryGroupReport, ReportRun
 from custom.ilsgateway.tanzania.warehouse_updater import populate_report_data
 from dimagi.utils.dates import force_to_datetime
+import settings
+from custom.logistics.tasks import stock_data_task
 
 
-# @periodic_task(run_every=timedelta(days=1), queue=getattr(settings, 'CELERY_PERIODIC_QUEUE', 'celery'))
+@periodic_task(run_every=crontab(hour="23", minute="55", day_of_week="*"),
+               queue=getattr(settings, 'CELERY_PERIODIC_QUEUE', 'celery'))
 def migration_task():
-    configs = ILSGatewayConfig.get_all_configs()
-    for config in configs:
+    for config in ILSGatewayConfig.get_all_steady_sync_configs():
         if config.enabled:
-            ils_bootstrap_domain(ILSGatewayAPI(config.domain, ILSGatewayEndpoint.from_config(config)))
+            endpoint = ILSGatewayEndpoint.from_config(config)
+            ils_bootstrap_domain(ILSGatewayAPI(config.domain, endpoint))
```

Comment:
We may want to split out the domain bootstrap from the data migration. I think some things don't need to run, though many do.

Can you confirm that the bootstrap task is smart enough to only find data since the previous run? I am looking into the code but haven't been able to verify that yet.